components/signupForm.tsx

```diff
@@ -181,14 +195,20 @@ export const SignupForm = () => {
             defaultValue={2}
             {...form.getInputProps('groupSize')}
             placeholder="Number of people"
-            label="Number of people"
+            label={
+              userType === 'refugee'
+                ? 'Number of people in your group'
+                : 'Number of people you can accomodate'
```

Comment:
something to look into might be if we have to collect information for all the refugees in the group to share with the organization, so then choosing a number > 1 could open up more fields to enter the others' info

Reply:
Definitely a good thought 🤔 something we should keep in mind once we start to develop relationships with local organizations, can morph the form into something that gives them exactly what they're looking for.